heads-or-tails.html

```diff
@@ -0,0 +1 @@
+![cointoss wireframe]
```

Comment:
I think this is why your Netlify preview is not working. We can check it out if you want later.

Reply:
Shoot! Yea, I would love to look at it later if you have the time.

Reply:
Here is the [[magic 8-ball URL](https://deploy-preview-1--hana-onboarding-fortune-teller.netlify.app/), but I think I broke my CI with my unfinished coin flip js..